README.md

```diff
@@ -55,6 +55,9 @@ Defaults to `false`
 add_filter( 'mercator.redirect.legacy.enabled', '__return_true' );
 ```
 
+**mercator.redirect.status.code**
+
+Defaults to 301
```

Comment:
Can you add a simple example too? Depending on any changes based on my feedback below that is.